utils.py

```diff
@@ -16,6 +16,14 @@ def trn_to_numpy(filename):
         return numpy_array.reshape(img.shape[0], img.shape[1])
 
 
+def numpy_to_trn(filename, mask):
+    filename_trn = "{}.trn".format(filename)
+    _, mask = cv.threshold(mask, 1, 1, cv.THRESH_BINARY)
+    f = open(filename_trn, 'w+')
```

Comment:
@Foxiy It would be great if you could use the following:

```python
with open(filename_trn, 'w+') as f:
    f.write(' '.join(str(e) for e in mask.flatten().tolist()))
```